src/main/webapp/app/localvc/repository-view/repository-view.component.ts

```
@@ -190,4 +193,27 @@ export class RepositoryViewComponent implements OnInit, OnDestroy {
             }),
         );
     }
+
+    private loadAuxiliaryRepository(exerciseId: number, auxiliaryRepositoryId: number) {
+        this.programmingExerciseService
+            .findWithAuxiliaryRepository(exerciseId)
+            .pipe(
+                tap((exerciseResponse) => {
+                    this.exercise = exerciseResponse.body!;
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)